apps/src/p5lab/constants.js

```diff
@@ -28,3 +28,9 @@ export const EMPTY_IMAGE =
   'data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==';
 
 export const PlayBehavior = utils.makeEnum('ALWAYS_PLAY', 'NEVER_PLAY');
+
+export const BLOCKLY_EVENTS_TO_LISTEN_FOR = [
+  'move', // Google Blockly - events listed here: https://github.com/google/blockly/blob/9089661f9d2209fe783a54c058e1ecaa7b8bcf7f/core/events/events.ts#L99-L129
+  'change', // Google Blockly
+  'blocklyBlockSpaceChange', // CDO Blockly
```

Comment:
Why do we need this? Based on the Slack thread, it sounds like this issue isn't happening on Sprite Lab with CDO Blockly, which is the only CDO Blockly P5Lab.

Reply:
If we don't include the CDO 'blocklyBlockSpaceChange' event, then the the change handler will not be added in CDO blockly so that preview will not work in sprite lab. This update adds the change handler for only this list of events which includes both CDO and Google blockly events.